host/onebox.go

```diff
@@ -424,7 +424,7 @@ func (c *cadenceImpl) startFrontend(hosts map[string][]string, startWG *sync.Wai
 	frontendConfig := frontend.NewConfig(dc, c.historyConfig.NumHistoryShards, c.workerConfig.EnableIndexer)
 	c.frontendHandler = frontend.NewWorkflowHandler(
 		c.frontEndService, frontendConfig, c.metadataMgr, c.historyMgr, c.historyV2Mgr,
-		c.visibilityMgr, kafkaProducer, params.BlobstoreClient, nil, nil)
+		c.visibilityMgr, kafkaProducer, params.BlobstoreClient, nil)
```

Comment:
Can you file a task to write integration using archiver?

Reply:
Task Created. #2187